code/modules/ghostroles/spawner/human/responseteams/vaurca/zora.dm

```diff
@@ -0,0 +1,35 @@
+/datum/ghostspawner/human/ert/zora
```

Comment:
Only the leadership role in a species restricted ERT may be whitelisted.
All the other roles must be accessible without a whitelist.

Reply:
`uses_species_whitelist = FALSE` does that not make them not require a whitelist

Reply:
Yeah - these all have species whitelists off except the TCAF one, which is playable as literally any species.

Reply:
> `uses_species_whitelist = FALSE` does that not make them not require a whitelist

Thats what I looked for.
And I completely missed it every time.